src/util.go

```diff
@@ -36,3 +36,11 @@ func Check(e error) {
 		os.Exit(1)
 	}
 }
+
+func GetUserName() (string, error) {
+	userName, err := GetGitConfigSetting("user.name")
+	if err != nil {
+		return "", err
+	}
+	return userName, nil
```

Comment:
There is
```go
if userName, err := GetGitConfigSetting("user.name"); err != nil {
  return "", err
} else {
  return userName, nil
}
```
as well, though I don't know if I like it. It kind of hides the condition.

Reply:
Is that common?

Reply:
I've definitely seen it and used it as well. But the PR is approved either way, so feel free to merge.